modules/consensus/accept_bench_test.go

```diff
@@ -0,0 +1,44 @@
+package consensus
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+)
+
+// benchmarkEmptyBlocks is a benchmark that mines many blocks, and
+// measures how long it takes to add them to the consensusset
+func benchmarkEmptyBlocks(bt *testing.B) error {
+	bt.StopTimer()
+
+	// The test dir will be reset each time the benchmark
+	// is done.
+	cst, err := createConsensusSetTester("BenchmarkEmptyBlocks")
+	if err != nil {
+		return errors.New("Error creating tester: " + err.Error())
+	}
+
+	for j := 0; j < bt.N; j++ {
+		b, _ := cst.miner.FindBlock()
+		bt.StartTimer()
+		err := cst.cs.AcceptBlock(b)
+		if err != nil {
+			errstr := fmt.Sprintf("Error accepting %d: %s", j, err.Error())
+			return errors.New(errstr)
+		}
+		cst.csUpdateWait()
+		bt.StopTimer()
+	}
+
+	return nil
+}
+
+// BenchmarkEmptyBlocks is a wrapper for benchmarkEmptyBlocks, which
+// handles error catching
+func BenchmarkEmptyBlocks(b *testing.B) {
```

Comment:
The current codebase is pretty inconsistent with this stuff, but the only reason you should have a lowercase function is if you plan on using it in future benchmarks.

But, because of the nature of benchmarking, I don't think that really makes sense. Just having everything be in one function is probably best.